omod/src/main/webapp/apiDocs.jsp

```diff
@@ -1,49 +1,109 @@
+    </symbol>
+
+    <symbol viewBox="0 0 20 20" id="locked">
+      <path d="M15.8 8H14V5.6C14 2.703 12.665 1 10 1 7.334 1 6 2.703 6 5.6V8H4c-.553 0-1 .646-1 1.199V17c0 .549.428 1.139.951 1.307l1.197.387C5.672 18.861 6.55 19 7.1 19h5.8c.549 0 1.428-.139 1.951-.307l1.196-.387c.524-.167.953-.757.953-1.306V9.199C17 8.646 16.352 8 15.8 8zM12 8H8V5.199C8 3.754 8.797 3 10 3c1.203 0 2 .754 2 2.199V8z"/>
+    </symbol>
+
+    <symbol viewBox="0 0 20 20" id="close">
+      <path d="M14.348 14.849c-.469.469-1.229.469-1.697 0L10 11.819l-2.651 3.029c-.469.469-1.229.469-1.697 0-.469-.469-.469-1.229 0-1.697l2.758-3.15-2.759-3.152c-.469-.469-.469-1.228 0-1.697.469-.469 1.228-.469 1.697 0L10 8.183l2.651-3.031c.469-.469 1.228-.469 1.697 0 .469.469.469 1.229 0 1.697l-2.758 3.152 2.758 3.15c.469.469.469 1.229 0 1.698z"/>
+    </symbol>
+
+    <symbol viewBox="0 0 20 20" id="large-arrow">
+      <path d="M13.25 10L6.109 2.58c-.268-.27-.268-.707 0-.979.268-.27.701-.27.969 0l7.83 7.908c.268.271.268.709 0 .979l-7.83 7.908c-.268.271-.701.27-.969 0-.268-.269-.268-.707 0-.979L13.25 10z"/>
+    </symbol>
 
+    <symbol viewBox="0 0 20 20" id="large-arrow-down">
+      <path d="M17.418 6.109c.272-.268.709-.268.979 0s.271.701 0 .969l-7.908 7.83c-.27.268-.707.268-.979 0l-7.908-7.83c-.27-.268-.27-.701 0-.969.271-.268.709-.268.979 0L10 13.25l7.418-7.141z"/>
+    </symbol>
+
+
+    <symbol viewBox="0 0 24 24" id="jump-to">
+      <path d="M19 7v4H5.83l3.58-3.59L8 6l-6 6 6 6 1.41-1.41L5.83 13H21V7z"/>
+    </symbol>
+
+    <symbol viewBox="0 0 24 24" id="expand">
+      <path d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"/>
+    </symbol>
+
+  </defs>
+</svg>
+
+<div id="swagger-ui"></div>
+
+<script src="${pageContext.request.contextPath}/moduleResources/webservices/rest/js/swagger-ui-3.0.10/swagger-ui-bundle.js"> </script>
+<script src="${pageContext.request.contextPath}/moduleResources/webservices/rest/js/swagger-ui-3.0.10/swagger-ui-standalone-preset.js"> </script>
 <openmrs:require privilege="Manage RESTWS" otherwise="/login.htm" redirect="/module/webservices/rest/settings.form" />
   <script type="text/javascript">
-    jQuery(document).ready(function() {
-      jQuery("#content").addClass("swagger-section ");
-        var swaggerUi = new SwaggerUi({
-          url:"${pageContext.request.contextPath}/module/webservices/rest/swagger.json",
-          dom_id:"swaggerDocumentation",
-          docExpansion: "none",
-          apisSorter: "alpha",
-          onFailure: function(data) {
-            console.log("Unable to Load SwaggerUI");
-            jQuery("#swaggerError").innerHTML = "Error Loading Swagger UI";
-          },
-          validatorUrl: null
-        });
-      swaggerUi.load();
-    });
+      window.onload = function() {
+          const ui = SwaggerUIBundle({
+              url: "${pageContext.request.contextPath}/module/webservices/rest/swagger.json",
```

Comment:
Is this also part of the upgrade?

Reply:
Yes. The newer versions use SwaggerUIBundle instead of SwaggerUi.